src/styles/blocks/price.css

```diff
@@ -0,0 +1,20 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-top: var(--margin-top-medium);
```

Comment:
The price seems to be a block. block cannot have external styles. follow BEM. you can handle this using a mix